src/api/views.py

```diff
@@ -496,4 +513,161 @@ def check_license(request):
+                serial.data, status=returnstatus
+                )
+        else:
+            return Response(
+                serializer.errors, status=status.HTTP_400_BAD_REQUEST
+                )
+
+
+def generateLicenseXml(licenseOsi, licenseIdentifier, licenseName, listVersionAdded, licenseSourceUrls, licenseHeader, licenseNotes, licenseText):
+    """ View for generating a spdx license xml
+    returns the license xml as a string
+    """
+    root = ET.Element("SPDXLicenseCollection", xmlns="http://www.spdx.org/license")
+    if licenseOsi=="Approved":
+        licenseOsi = "true"
+    else:
+        licenseOsi = "false"
+    license = ET.SubElement(root, "license", isOsiApproved=licenseOsi, licenseId=licenseIdentifier, name=licenseName, listVersionAdded=listVersionAdded)
+    crossRefs = ET.SubElement(license, "crossRefs")
+    for sourceUrl in licenseSourceUrls:
+        ET.SubElement(crossRefs, "crossRef").text = sourceUrl
+    ET.SubElement(license, "standardLicenseHeader").text = licenseHeader
+    ET.SubElement(license, "notes").text = licenseNotes
+    licenseTextElement = ET.SubElement(license, "text")
+    licenseLines = licenseText.replace('\r','').split('\n')
+    for licenseLine in licenseLines:
+        ET.SubElement(licenseTextElement, "p").text = licenseLine
+    xmlString = ET.tostring(root, method='xml').replace('>','>\n')
+    return xmlString
+
+
+def createIssue(licenseAuthorName, licenseName, licenseIdentifier, licenseComments, licenseSourceUrls, licenseHeader, licenseOsi, token, urlType):
+    """ View for creating an GitbHub issue
+    when submitting a new license request
+    """
+    body = '**1.** License Name: ' + licenseName + '\n**2.** Short identifier: ' + licenseIdentifier + '\n**3.** License Author or steward: ' + str(licenseAuthorName)+ '\n**4.** Comments: ' + str(licenseComments) + '\n**5.** Standard License Header: ' + str(licenseHeader) + '\n**6.** URL: '
+    for url in licenseSourceUrls:
+        body += url
+        body += '\n'
+    body += '**7.** OSI Status: ' + licenseOsi
+    title = 'New license request: ' + licenseIdentifier + ' [SPDX-Online-Tools]'
+    payload = {'title' : title, 'body': body, 'labels': ['new license/exception request']}
+    headers = {'Authorization': 'token ' + token}
+    url = TYPE_TO_URL[urlType]
+    r = post(url, data=dumps(payload), headers=headers)
+    return r.status_code
+
+
+def validate_license_fields(licenseName, licenseIdentifier):
+    """ Validate the licenseName and licenseIdentifier
+    when submitting a new license
+    """
+    no_comma_match = bool(re.compile(r'^((?!,).)*$').match(licenseName))
+    no_version_match = bool(re.compile(r'^((?!version).)*$').match(licenseName))
+    lower_v_match = bool(re.compile(r'^((?!v\.|v\s).)*$').match(licenseName))
+    the_match = bool(re.compile(r'^(?!the|The.*$).*$').match(licenseName))
+
+    if not no_comma_match:
+        return 'No commas in the fullname of license or exception.'
```

Comment:
The message can be changed to `No commas allowed in the ...`, this would make the message more clear.

Reply:
This was the same message as mentioned in the SPDX website about the description of the fields. Although, I agree it makes the message more clear to understand.